src/components/categories/Categories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+const Categories = () => {
+  const dispatch = useDispatch();
+  const showTextUnderConstruction = () => {
+    dispatch({ type: 'ACTIVE' });
```

Comment:
- [ ] Kindly be noticed that the `'ACTIVE'` isn't the actual value that you should send to the reducer please change it to the actual value which is `'redux/categories/ACTIVE''`.